lib/htmlValidator.js

```diff
@@ -138,9 +138,39 @@ module.exports = function (app, callback) {
     let render = app.response.render
 
     app.response.render = function (view, model, callback) {
-      if (model[modelException]) {
-        this.req.headers[headerException] = true
+      if (!model) {
+        model = {}
       }
+
+      let requestHeaders = this.req.headers
+      if (modelException) {
+        if (typeof modelException === 'string') {
+          if (model[modelException]) {
+            applyHeaderExceptions()
```

Comment:
Simple refactor that could possibly make the code coverage aspect easier:

Rather than testing exceptions for both `string` and `array` config examples, coerce your `string` scenario into an array, then you only have to run the match logic and execute the header exception function in one place.

You could do this by caching your exceptions in a new variable that is initialized as an empty array, then either pushing the string into it if the param is a string, or just setting it to whatever the array is in the case of it being configured as an array, then run your model/header exception loop as normal.